src/document/utils.js

```diff
@@ -158,37 +158,34 @@ function mapDoc(doc, cb) {
   }
 }
 
-function findInDoc(doc, fn, defaultValue) {
-  let result = defaultValue;
-  let hasStopped = false;
```

Comment:
This is useless, `onEnter` returns `false` will stop traverse, must left over during refactoring https://github.com/prettier/prettier/pull/442

Reply:
Maybe I'm wrong, adding test.

Reply:
return false on `onEnter` doesn't stop traverse. https://github.com/prettier/prettier/pull/13349/commits/e3ff2edc6f778797659dbd8a323a4d26a7be3405